kroxylicious-filters/kroxylicious-multitenant/pom.xml

```diff
@@ -114,6 +119,25 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-enforcer-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>filters-use-public-api</id>
+                        <configuration>
+                            <rules>
+                                <bannedDependencies>
+                                    <includes>
+                                        <!-- runtime is a test dependency, enforcer cannot take scope into account-->
```

Comment:
hmm that sucks a bit.

Reply:
gah maybe I've misread it, they have examples like

```
org.apache.maven:*:*:jar:test
*:*:*:jar:compile:tests
```